includes/class-convertkit-resource-forms.php

```diff
@@ -496,7 +497,8 @@ public function get_html( $id, $post_id = 0 ) {
 					/* translators: ConvertKit Form ID */
 					__( 'Kit Form ID %s does not exist on Kit.', 'convertkit' ),
 					$id
-				)
+				),
+				404
```

Comment:
Provides context to the error i.e. a Kit Form ID was specified that either does not exist in the cached resources, is an ID for a legacy form, or truly doesn't exist (the user entered the wrong ID in the shortcode).